packages/web-client/app/components/card-pay/create-merchant-workflow/prepaid-card-choice/index.hbs

```diff
@@ -27,7 +28,29 @@
       />
     </CardPay::LabeledValue>
     <CardPay::LabeledValue @label="Pay Using Prepaid Card" class="prepaid-card-choice__field-label">
-      {{!-- TODO: card picker --}}
+      {{#if this.selectedPrepaidCard}}
+        <div class="prepaid-card-choice__change-card">
+          <CardPay::CardPicker::CardOption
+            @card={{this.selectedPrepaidCard}}
+            data-test-prepaid-card-choice-selected-card
+          />
+          {{#if (and (not @isComplete) (not-eq this.creationState "in-progress"))}}
```

Comment:
There’s no real standard on this but to me this is close to the point where it makes sense to extract as a getter in the component, something like `{{#if this.showSelectedPrepaidCard}}`